cli/src/gas/gas_add_budget.rs

```diff
@@ -0,0 +1,86 @@
+use {
+    crate::{command_title, display::json_output, loading, prelude::*, sui::*},
+    nexus_sdk::transactions::gas,
+};
+
+/// Upload `coin` as a gas budget for the Nexus workflow.
+pub(crate) async fn add_gas_budget(
+    coin: sui::ObjectID,
+    sui_gas_coin: Option<sui::ObjectID>,
+    sui_gas_budget: u64,
+) -> AnyResult<(), NexusCliError> {
+    command_title!("Adding '{coin}' as gas budget for Nexus");
+
+    // Load CLI configuration.
+    let conf = CliConf::load().await.unwrap_or_else(|_| CliConf::default());
```

Comment:
This pattern is repeated across several commands. I'd suggest moving it into the `CliConf` itself as `CliConf::load_or_default()`, and also simplify it (no need for the lazy evaluation here):

```suggestion
    let conf = CliConf::load().await.unwrap_or(CliConf::default());
```

Reply:
using `let conf = CliConf::load().await.unwrap_or_default();`